Apply: Francis feels frustrated post (#974)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md

---
title: When Franics felt frustrated
description: We developed a story about subject knowledge enhancement courses. The purpose of the story is to show how we can empathise with users by capturing their pain points in an engaging way.
date: 2023-07-12
tags:
  - subject knowledge enhancements
ogImage:
  src: francis-cover-image.png
  alt: "Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews."
---

![Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews.](francis-cover-image.png)

> Previously, we designed guidance to tell candidates they have to do a [subject knowledge enhancement (SKE) course](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.
> 
> We tested these designs and learnt a lot about SKE courses. Some of the findings we uncovered were about the wider SKE policy and do not fit with Apply for teacher training (Apply).

Reviewer: I wonder if this opening para could be a bit more punchy, and summarise (or at least hint at) the issues themselves? Might also be good to briefly summarise what the research consisted of?

This suggestion isn’t quite right either, but might be an approach to try?

```suggestion
We did some research around subject knowledge enhancement (SKE) courses, as part of [designing guidance telling candidates when they have to do these](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.

We usability tested the designs with X participants, and also sent a survey to X people who had previously received an offer with a SKE course condition attached.

This research revealed some wider issues with how users choose a course, as well as the process of applying to a a course and receiving the funding, and the content of the course itself.
```

We wanted to capture these findings in a way that would help people empathise with candidates and providers. So, we wrote a story....

> We’ve added the story as a design history to share with other teams. It's purpose is to think about different ways to present research and show empathy beyond individual services.

Reviewer: I feel like this paragraph could be dropped?  The previous paragraph has a nice lead-in to the story already.

```suggestion
```

They ask Francis to send her degree certificate and proof of her home fee status.  

“Not again” says Francis.

![Screenshot showing a page with the old 'Interview needs' question with information about COVID-19 and how providers are not flexible with interview dates. The questions asl reads, 'Do you have any interview needs?'.](another-form.png)

## Starting the SKE course

Francis starts her SKE course online. She does it alone and mostly in the evenings after work.  

 There’s a lot to do and she must write long reflection essays at the end of every week.

Francis feels frustrated that she’s not spending enough time doing practical chemistry exercises.

![Screenshot showing a page with the old 'Interview needs' question with information about COVID-19 and how providers are not flexible with interview dates. The questions asl reads, 'Do you have any interview needs?'.](studying-at-night.png)

## Surprise payment

Randomly, ExtrA Learning Trust asks for Francis’ bank details, she’s not sure why.  

Francis finds out she will get £175 a week to do her SKE course.  

It’s less than she earns as a waiter and she’s halfway through her SKE course already.  

“Bit late, but ok”

> Francis fills out another form to provider her bak details.

Reviewer: ```suggestion
Francis fills out another form to provider her bank details.
```

![Illustration of a woman who looks happy. Around her head are icons of different currencies like the dollar, pound and euro.](bursary.png)

## Time to start teacher training

Just before her teacher training starts, Francis goes to Florence! She’s saved up a whole year for this.  

Francis falls behind on her SKE course work and must finish it during the first 2 weeks of her teacher training.  

Suddenly, the SKE bursary payments stop. She didn’t know you can’t get the bursary if it overlaps with teacher training.  

 Francis feels frustrated!

![Illustration of a woman looking frustrated with her hands on her head. Around her head are icons of exclamation marks and question marks to show her frustration.](francis-confused.png)

Francis makes some friends on her teacher training course. They also had to do an SKE course.  

Everyone has different things to say about their SKE course – from different lengths to different content.  

Some thought they were fabulous, some thought they were frightening.

Francis loves her teacher training course, but she’s relearning things she did in her SKE course.  

>  Francis feels frustrated (again).

Reviewer 2: Is there a solution to Francis problems?
